src/js/libcs/List.js

```diff
@@ -2511,3 +2511,17 @@ List.ofGeos = function(geos) {
         };
     }));
 };
+
+List.getRandRealVec = function(min, max) {
+    var mmin = min || 0;
+    var mmax = max || 1;
+
+    return List.turnIntoCSList([CSNumber.getRandReal(mmin, mmax), CSNumber.getRandReal(mmin, mmax), CSNumber.getRandReal(mmin, mmax)]);
```

Comment:
I'd prefer to see this wrapped to stay within 80 columns. One line per `getRandReal` invocation, plus one for the opening and one for the closing stuff around them. This also makes it more obvious that you are doing the exactly same invocation three times.

I wonder whether the length of the vector, i.e. 3, should be encoded in the name of the function. Analogous to `det3` and `adjoin3`. I think that would make sense.